src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -620,6 +625,29 @@ export class CancerSummaryChart extends React.Component<
         });
     }
 
+    @autobind
+    private getData(dataType: DataType) {
+        return tsvFormat(this.convertDataToDownloadData(this.props.data));
+    }
+
+    @autobind
+    private convertDataToDownloadData(data: any[][]): any[] {
+        const downloadData: any[] = [];
+
+        data.forEach(elementar => {
+            elementar.forEach(element => {
+                if (element.y != 0) {
+                    downloadData.push({
+                        x: element.x,
```

Comment:
We should improve these column names.    The identity of both x and y depend on user selection in the controls at top.   Basically, 
x = Cancer Type || Cancer Type Detailed || Cancer Study
y = Alteration Frequency || Absolute Count

Reply:
this functionality has been added